### ISSUE TITLE
pyVoIP.SIP.error.SIPParseError: Regex failed to match To/From.

### ISSUE THREAD

**Issue by VadyChel** [tayler6000/pyVoIP#154]
I use binotel sip server and 2.0.0-dev version of pyvoip
Error:
data='"sipvicious"<sip:100@1.1.1.1>' type(match)=<class 'NoneType'>

**Comment by tayler6000** [comment]
Thanks for creating this issue!

I'd have to check the RFC, but I'm 95% sure SIP Viscous has a malformed header there.  But considering it's a penetration testing tool, it might be intentional.

Does anyone have an opinion on if I should cover this case or not?

**Comment by jchatin** [comment]
Hi Tayler,

I found different issues on headers in the code of pyVoIP.
Parsing the To and From headers are not done correctly :
Using strip method to parse To or From header cannot work if header is formated like this "PhoneNumber <phonenumber@sip.server.org:5060>" :
Original code:
``` 
t = request.headers["To"]["raw"].strip("<").strip(">")
``` 
Working code:
``` 
t = request.headers["To"]["raw"]
t = re.sub("(.*<)(.*)(>.*)","\\2",t)
```

Also branch, tags and cseq were not generated correctly as per the RFC.
I was able to fix these issues by reading the SIP RFC and network Frame analysis with tcpdump.
